com.unity.cinemachine/Editor/Upgrader/CinemachineUpgradeManager.cs

```diff
@@ -392,7 +192,8 @@ void UpgradeAllScenes(Dictionary<int, List<ConversionLink>> conversionLinksPerSc
                 // GML todo: do we need to do this recursively for child GameObjects?
```

Comment:
What about this?  Did you test with a prefab that has a deep structure?

Reply:
Tested it, no problem.